tests/e2e/v3_curl_test.go

```diff
@@ -244,7 +245,7 @@ func testV3CurlAuth(cx ctlCtx) {
 			lineFunc   = func(txt string) bool { return true }
 		)
 
-		cmdArgs = e2e.CURLPrefixArgsCluster(cx.epc, "POST", e2e.CURLReq{Endpoint: path.Join(p, "/auth/authenticate"), Value: string(authreq)})
+		cmdArgs = e2e.CURLPrefixArgsCluster(cx.epc.Cfg, cx.epc.Procs[rand.Intn(cx.epc.Cfg.ClusterSize)], "POST", e2e.CURLReq{Endpoint: path.Join(p, "/auth/authenticate"), Value: string(authreq)})
```

Comment:
Where this change from? Don't see it in https://github.com/etcd-io/etcd/pull/13405/files nor https://github.com/etcd-io/etcd/pull/16457/files

Reply:
it's part of https://github.com/etcd-io/etcd/commit/75449c075df341e580acf2f2333ec854b4c75826. could not find the PR for it.